packages/core/types/index.js

```diff
@@ -173,7 +173,8 @@ export type ParcelOptions = {|
   logLevel: LogLevel,
   rootDir: FilePath,
   targets: Array<Target>,
-  projectRoot: FilePath
+  projectRoot: FilePath,
+  lockFile?: FilePath
```

Comment:
This should probably be `lockFile: ?FilePath`, as we produce this object and can guarantee the key.